edit portfolio.html

```diff
@@ -0,0 +1,163 @@
+              required
+            />
+            <label for="startDate"></label>
+            <input type="date" id="startDate" required />
+            <label for="endDate"></label>
+            <input type="date" id="endDate" required />
+            <label for="description"></label>
+            <textarea
+              id="description"
+              rows="4"
+              placeholder="Description"
+              required
+            ></textarea>
+            <button id="saveButton">Save</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="side-bar">
+        <div class="avatar">
+          <img src="assets/avatar.jpeg" />
+          <h2>Mishal Firdous</h2>
+          <ul class="a">
+            <li>
+              <label for="file">Wordpress, Wix, Shopify </label>
+              <progress id="file" value="90" max="100">90%</progress>
+            </li>
+
+            <li>
+              <label for="file">HTML5 ,CSS3, Bootstrap</label>
+              <progress id="file" value="70" max="100">32%</progress>
+            </li>
+
+            <li>
+              <label for="file">Adobe Photo Shop </label>
+              <progress id="file" value="30" max="100">32%</progress>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <script>
+      document.addEventListener("DOMContentLoaded", function () {
+        const addButton = document.getElementById("addButton");
+        const experienceForm = document.getElementById("experienceForm");
+        const saveButton = document.getElementById("saveButton");
+        const experienceList = document.getElementById("experienceList");
+        //store data in localStorage
+        const dataValue =
+          JSON.parse(localStorage.getItem("experiencedata")) || [];
+
+        addButton.addEventListener("click", function () {
+          experienceForm.style.display = "block";
+        });
+
+        saveButton.addEventListener("click", function () {
+          const companyName = document.getElementById("companyName").value;
+          const startDate = document.getElementById("startDate").value;
+          const endDate = document.getElementById("endDate").value;
+          const description = document.getElementById("description").value;
+          const experienceData = {
+            companyName,
+            startDate,
+            endDate,
+            description,
+          };
+
+          dataValue.push(JSON.stringify(experienceData));
+          localStorage.setItem("experiencedata", JSON.stringify(dataValue));
+          
+          displayLocalStorage();
+
+          document.getElementById("companyName").value = "";
+          document.getElementById("startDate").value = "";
+          document.getElementById("endDate").value = "";
+          document.getElementById("description").value = "";
+
+          experienceForm.style.display = "none";
+        });
+      });
+      function displayLocalStorage() {
```

Comment:
Name the function as DisplayLocalStorageData or something that makes more sense and move it to separate js file and use it as it is duplicated in the two files.

Reply:
resolved!